sdk/tools/kbdtool/output.c

```diff
@@ -714,6 +714,7 @@ kbd_c(IN ULONG StateCount,
         if ((Layout->Entry[i].ScanCode & 0xFF00) == 0xE000)
         {
             /* Which are valid */
+            // FIXME: This check is always true.
```

Comment:
How so? If someone makes a layout with a scancode value `0xE0FF` ?
(And same question for the similar fixme below.)

Reply:
After line 714 check, we know `ScanCode >= 0xE000`, hence it cannot be `0x00FF`.

Reply:
Unless the purpose of the second test was to check the lowest byte of the scan code? (And they forgot the `& 0xFF`)